src/common/libflux/event.c

```diff
@@ -22,6 +22,68 @@
 #include "rpc.h"
 #include "message.h"
 
+flux_future_t *flux_event_subscribe_ex (flux_t *h,
+                                        const char *topic,
+                                        int flags)
+{
+    if (!h || !topic) {
+        errno = EINVAL;
+        return NULL;
+    }
```

Comment:
If `flux_rpc_pack(3)` checks `h` and `topic` then this check could be dropped.

Reply:
h is checked but topic is not (it's not the rpc topic but part of the payload).
i'll drop the h check.